engineering-feedback/feedback-examples.md

```diff
@@ -0,0 +1,46 @@
+# Engineering Feedback Examples
+
+The following are real world examples of Engineering Feedback that have led to product improvements and unblocked customers.
+
+## Windows Server Container support for Azure Kubernetes Service
+
+The Azure Kubernetes Service should have first class Windows container support so solutions that require Windows workloads can be deployed on a wildly popular container orchestration platform. The need was to be able to deploy Windows Server containers on AKS the managed Azure Kubernetes Service. According to [this FAQ](https://docs.microsoft.com/en-us/azure/aks/faq#can-i-run-windows-server-containers-on-aks) (and in parallel confirmation) it is not available yet.
+
+ We tried to deploy anyway as a test, and it did not work – the deployment would be pending without success.
+
+More than a dozen large partners/customers are blocked in deploying Windows workloads to AKS due to a lack of support for Windows Server containers. They need this feature so solutions requiring Windows workloads can be deployed to this popular container orchestration platform.
+
+We are seeing an emergence of companies beginning to try Windows containers as an option to move their Windows workloads to the cloud.  Gartner is claiming that 80% of enterprise apps run on Windows. Containers have become the de facto deployment mechanism in the industry, and deployment consistency and speed are a few of the important factors companies are looking for. Enabling Windows applications and ensuring that developers have a good experience when moving their workloads to Azure via Windows containers is key to keeping existing Windows customers within the Azure ecosystem and driving Azure adoption for new workloads.
```

Comment:
Is this information really pertinent to the feedback?  In my opinion, it doesn't convey why this specific customer needed Windows container support.  It conveys a general sentiment on why Windows containers are needed, but I feel it lacks specifics.